### PR TITLE
exiv2: update to 0.27.4

### PR DESCRIPTION
**Pull request by kmilos**

### CODE REVIEW

File: mingw-w64-exiv2/PKGBUILD [hunk 1]
```diff
@@ -1,61 +1,64 @@
...
-license=("GPL")
+url='https://exiv2.org/'
+license=('GPL-2.0')
 makedepends=("${MINGW_PACKAGE_PREFIX}-gcc"
-             "${MINGW_PACKAGE_PREFIX}-pkg-config"
-             "${MINGW_PACKAGE_PREFIX}-libtool"
-             "${MINGW_PACKAGE_PREFIX}-cmake")
-depends=("${MINGW_PACKAGE_PREFIX}-expat"
-         "${MINGW_PACKAGE_PREFIX}-curl"
-         "${MINGW_PACKAGE_PREFIX}-libiconv"
```
Comment (lazka) [inline mingw-w64-exiv2/PKGBUILD:20]: why did you remove expand and libiconv?

Comment (kmilos) [reply mingw-w64-exiv2/PKGBUILD:20]: Replaced by gettext (+ enabled translations) that depends on them anyway

Comment (lazka) [reply mingw-w64-exiv2/PKGBUILD:20]: Not specifying direct dependencies makes it hard/impossible to know what needs to be rebuilt on an ABI break.

Comment (kmilos) [reply mingw-w64-exiv2/PKGBUILD:20]: Ok, reverted
